open-craft/edx-platform
# [BB-2837] Update latest reviewed Arabic translations from Transifex

This PR adds the latest reviewed Arabic Translations from Transifex.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-2837

~~**Discussions**: ~~

**Dependencies**: None

~~**Screenshots**:~~

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: None

**Testing instructions**:

1. Pull this PR in your local devstack.
2. Make Arabic ``ar`` as the default language from ``/edx/etc/lms.py``
3. Check if the latest updates from Transifex has applied.

**Author notes and concerns**:
N/A

**Reviewers**

**Settings**
```yaml
```
